extra-network/clash-premium/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=clash-premium
+PKGSEC=net
+PKGDEP="glibc"
+PKGDES="Close-source version of Clash with additional features."
+ABSPLITDBG=0
+PKGCONFL=clash
+FAIL_ARCH="!(amd64|arm64|loongson3)"
```

Comment:
Did the upstream not end up offering POWER binaries?

Reply:
https://github.com/Dreamacro/clash/issues/2032#issuecomment-1075973637
Awaiting for response